lib/main.dart

```diff
@@ -70,9 +76,14 @@ class _MyAppState extends State<MyApp> {
 
     // create services to pass to app
     _navigation = widget._navigation ?? Navigation();
+
+    final geolocator = GeolocatorPlatform.instance;
+    _locationService = widget._locationService ?? LocationService(geolocator);
```

Comment:
could you do this in a single line?  eg. 

`_locationService = widget._locationService ?? LocationService(GeolocatorPlatform.instance; );`